lib/screens/forgot_password/forgot_password_repo.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:math';
+
+import 'package:ecellapp/core/res/errors.dart';
+
+abstract class ForgotPasswordRepository {
+  Future<int> generateOTP(String email);
+
+  Future<bool> verifyOTP(String x);
```

Comment:
give better variable names.
```suggestion
  Future<bool> verifyOTP(String otp);
```